s3-backup.sh

```diff
@@ -43,7 +43,7 @@ mc_backup () {
     tmpdir=$(mktemp -d)
     chmod 700 $tmpdir
     BACKUP=$tmpdir/backup-$(date +%Y%m%d-%H%M).tar
```

Comment:
Should the filename pattern now be `backup-DATE.tar.xz.enc` or even just `backup-DATE.enc`?

Reply:
For sanity, seems to make sense having it all (tar.xz.enc) or nothing (enc). Probably best with the extension, then tar's `--auto-compress` will do the right thing for any changes down the road.